### ISSUE TITLE
ModuleNotFoundError: No module named 'agents'

### ISSUE THREAD

**Issue by starlighttt123** [Thinklab-SJTU/Bench2Drive#90]
Hi, I follow the readme to eval, but when running `python tools/ability_benchmark.py -r /home/user/new_Bench2Drive/Bench2Drive/vad_b2d_traj/merged.json`

It occurs:
```
Traceback (most recent call last):
  File "tools/ability_benchmark.py", line 5, in <module>
    from agents.navigation.global_route_planner import GlobalRoutePlanner
ModuleNotFoundError: No module named 'agents'
```
and  #12  doesn't seem to help
Thanks for your reply!

**Comment by jayyoung0802** [comment]
Need set your carla path. [set carla path](https://github.com/Thinklab-SJTU/Bench2Drive/blob/main/leaderboard/scripts/run_evaluation.sh#L3)

**Comment by jiangxb98** [comment]
Hello,I also encountered this problem. Could you please tell me how you solved it?

**Comment by jiangxb98** [comment]
I have solved this problem floow the issue #12
